### PR TITLE
amixer: add amixer files for LNL platforms

### PR DESCRIPTION
**Pull request by keqiaozhang**

### CODE REVIEW

File: alsa_settings/LNLM_SDW_AIOC.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# enable headset playback
+amixer -c sofsoundwire cset name='Headphone Switch' on
+amixer -c sofsoundwire cset name='rt711 FU05 Playback Volume' 80
+
+# enable headset capture
+amixer -c sofsoundwire cset name='Headset Mic Switch' on
+amixer -c sofsoundwire cset name='rt711 FU0F Capture Switch' on
+amixer -c sofsoundwire cset name='rt711 FU1E Capture Switch' on
+amixer -c sofsoundwire cset name='rt711 FU0F Capture Volume' 30
+amixer -c sofsoundwire cset name='rt711 FU1E Capture Volume' 30
```
Comment (plbossart) [inline alsa_settings/LNLM_SDW_AIOC.sh:12]: what about LNLM_SDW_RVP without AIOC? That's what I use so far :-)

**Review by marc-hb — CHANGES_REQUESTED** [review]
Just like #1124, Jenkins did not pick that up. Blocking merge until it gets tested.

**Comment by keqiaozhang** [comment]
SOFCI TEST

**Comment by keqiaozhang** [comment]
SOFCI TEST

**Comment by keqiaozhang** [comment]
SOFCI TEST

**Comment by keqiaozhang** [comment]
SOFCI TEST

**Comment by keqiaozhang** [comment]
SOFCI TEST

**Comment by keqiaozhang** [comment]
SOFCI TEST

**Comment by keqiaozhang** [comment]
SOFCI TEST

File: alsa_settings/LNLM_SDW_AIOC.sh (hunk 1 above)
Comment (keqiaozhang) [reply alsa_settings/LNLM_SDW_AIOC.sh:12]: LNLM_SDW_RVP is not our target test config. We will switch all LNLM_SDW_RVP to LNLM_SDW_AIOC.

**Review by aiChaoSONG — APPROVED** [review]

**Pull request merged**